### PR TITLE
docs: hammerjs using systemjs.config

### PR DESCRIPTION
**Pull request by Longfld**
To re-do and replace #7918

### CODE REVIEW

File: guides/getting-started.md [hunk 1]
```diff
@@ -189,7 +189,11 @@ System.config({
     '@angular/cdk/platform': 'npm:@angular/cdk/bundles/cdk-platform.umd.js',
     '@angular/cdk/a11y': 'npm:@angular/cdk/bundles/cdk-a11y.umd.js',
     // ...
-  }
+    'hammerjs': 'npm:hammerjs',
+  },
+  packages: {
```
Comment (crisbeto) [inline guides/getting-started.md]: The indentation here is inconsistent with the rest of the code snippet.

File: guides/getting-started.md [hunk 2]
```diff
@@ -189,7 +189,11 @@ System.config({
...
+      hammerjs:  { main: './hammer.js', defaultExtension: 'js'}
```
Comment (crisbeto) [inline guides/getting-started.md]: You've got an extra space before and after the curly brace. Also shouldn't this use the minified version of the file?

Comment (Longfld) [reply guides/getting-started.md]: systemjs, normally used as development. And if someone wants to bring it in production, I will assume they should know how to config. it.
But if @crisbeto believes it is better to have minified version, please let me know.

Comment (crisbeto) [reply guides/getting-started.md]: SystemJS does have a bundler that can be used for production, but I'm fine with continuing to use the non-minified version. The comment from above about the indentation still stands though.

Comment (Longfld) [reply guides/getting-started.md]: @crisbeto  somehow, I cannot push my changes from my fork anymore.  Do you want to take over this pull and we can have job done?
https://github.com/Longfld/material2/blob/master/guides/getting-started.md
For minified version:
 hammerjs: {main: './hammer.min.js',defaultExtension: 'js'}

Comment (crisbeto) [reply guides/getting-started.md]: That makes it trickier with the CLA. We're not in a rush to get this one in anyway. You might be getting blocked from pushing to your fork if your git email is out of sync with the one from GitHub.

Comment (Longfld) [reply guides/getting-started.md]: @crisbeto thanks enlighten me! I can push from my fork now. waiting for your further comments

Comment (crisbeto) [reply guides/getting-started.md]: Sorry for being nitpicky, but it looks like the indentation still doesn't line up with the rest of the file.

Comment (Longfld) [reply guides/getting-started.md]: You are right, it is me, got difficult to commit in,  please see #10711

**Comment by Longfld** [comment]
Replace by #10711

**Pull request closed without merging**